### ISSUE TITLE
2.0.7 Date Abbreviations shortcuts not being expanded

### ISSUE THREAD

**Issue by sleebooth** [fisharebest/webtrees#3453]
None of the usual shortcuts for dates are being expanded

e.g. Q1 1844 should become BET JAN 1844 AND MAR 1844 but it becomes Q 1 1844 when the event is saved.
1841-1851 remains as 1841 under the date field but the date becomes 1841 1851

**Comment by sleebooth** [comment]
Tested on Demo using
Google Chrome Version 84.0.4147.105 (Official Build) (64-bit)

**Comment by sleebooth** [comment]
Same as https://github.com/fisharebest/webtrees/issues/3439

Fixed in https://github.com/fisharebest/webtrees/commit/4ad647d513adf58792669ecfe5084f4d217b9794

However, the file public/resources/js/webtrees.js did not exist on my site, so I can only assume that the automatic update to 2.07, although it said it completed, did not include that file. (In order to get the update to work, I had to keep trying several times, otherwise I was getting a cURL timeout. Not sure if this might be connected)

By updating / inserting the two files, the bug is fixed.
